server/server.go

```diff
@@ -769,3 +845,46 @@ func (s *Server) startGoroutine(f func()) {
 		s.goroutineWait.Done()
 	}()
 }
+
+// publishEvent publishes an event on the activity stream.
+func (s *Server) publishEvent(streamEvent client.ActivityStreamEvent) error {
```

Comment:
I think this should have a less ambiguous name - eg `publishActivityEvent()`

Reply:
Yes. Should be fixed in 65d1e5e.